eng/testing/tests.targets

```diff
@@ -115,6 +118,69 @@
     <Error Condition="'$(TestRunExitCode)' != '0'" Text="$(TestRunErrorMessage)" />
   </Target>
 
+  <!-- Generate a self-contained app bundle for iOS with tests.
+       This target is executed once build is done for a test lib (after CopyFilesToOutputDirectory target) -->
+  <UsingTask TaskName="AppleAppBuilderTask" 
+             AssemblyFile="$(ArtifactsObjDir)mono\AppleAppBuilder\$(TargetArchitecture)\$(Configuration)\AppleAppBuilder.dll" />
+  <Target Condition="'$(TargetOS)' == 'iOS'" Name="BundleTestAppleApp" AfterTargets="CopyFilesToOutputDirectory">
+    <PropertyGroup>
+      <RuntimePackDir>$(ArtifactsDir)bin\lib-runtime-packs\runtimes\ios-$(TargetArchitecture)</RuntimePackDir>
+      <BundleDir>$(OutDir)\Bundle</BundleDir>
+      <AppleTestRunner>$(RepoRoot)\src\mono\msbuild\AppleTestRunner\bin</AppleTestRunner>
+    </PropertyGroup>
+    <!-- We need to copy additional stuff into $(OutDir)\Bundle
+         1) The whole BCL
+         2) Test Runner (with xharness client-side lib)
+      -->
+    <ItemGroup>
+      <TestBinaries Include="$(OutDir)\*.*"/>
+      <AppleTestRunnerBinaries Include="$(AppleTestRunner)\*.*" />
+      <BclBinaries Include="$(RuntimePackDir)\lib\$(NetCoreAppCurrent)\*.*" Exclude="$(RuntimePackDir)\lib\$(NetCoreAppCurrent)\System.Runtime.WindowsRuntime.dll" />
+      <BclBinaries Include="$(RuntimePackDir)\native\*.*" />
+    </ItemGroup>
+    <Error Condition="!Exists('$(AppleTestRunner)')" Text="AppleTestRunner=$(AppleTestRunner) doesn't exist" />
+    <Error Condition="!Exists('$(RuntimePackDir)')" Text="RuntimePackDir=$(RuntimePackDir) doesn't exist" />
+    <RemoveDir Directories="$(BundleDir)" />
+    <Copy SourceFiles="@(TestBinaries)" DestinationFolder="$(BundleDir)" SkipUnchangedFiles="true"/>
+    <Copy SourceFiles="@(AppleTestRunnerBinaries)" DestinationFolder="$(BundleDir)\%(RecursiveDir)" SkipUnchangedFiles="true"/>
+    <Copy SourceFiles="@(BclBinaries)" DestinationFolder="$(BundleDir)\%(RecursiveDir)" SkipUnchangedFiles="true"/>
+
+    <!-- xharness expects 'xunit-excludes.txt' with xunit categories to ignore in the bundle -->
+    <ItemGroup>
+      <XunitIgnoreCategories Include="category=OuterLoop"/>
+      <XunitIgnoreCategories Include="category=failing"/>
+      <XunitIgnoreCategories Include="category=nonnetcoreapptests"/>
+      <XunitIgnoreCategories Include="category=nonosxtests"/>
+      <XunitIgnoreCategories Include="category=nonmonotests"/>
+    </ItemGroup>
```

Comment:
instead of this can you reuse https://github.com/dotnet/runtime/blob/b1b08fc64e6b0aab4f52167191d2d21d439ac02a/eng/testing/xunit/xunit.console.targets#L16-L17 ?

Reply:
You don't need the non...tests traits anymore.

Reply:
@ViktorHofer we'd like to merge it as is to let others to play with it and test if you don't mind, I'll remove this stuff in a follow-up PR (later today or tomorrow)

Reply:
sure, sounds good.